packages/elements/src/components/base/_typography.scss

```diff
@@ -0,0 +1,73 @@
+// ##################################
+// Font and typography
+// ##################################
+
+// => Font Family
+
+$_font-families: (
+  sans-serif: unquote("'Gilroy', Verdana, sans-serif"),
```

Comment:
Where did you get these font-families from?
Did you c&p these by mistake?
We used `'Lato', Helvetica, Verdana, sans-serif` before.

Reply:
True that. Fixed it, and renamed the mixins of spinner to the private`_` syntax (thanks @AlessaRad)